unsegregated/_act/prompts.md

```
@@ -1,5 +1,14 @@
 ## Prompts
 
+Registration Client UI Spec
+- Read through the Registration Client UI Spec attached here
+- Also read through Spec from https://docs.mosip.io/1.1.5/modules/registration-client/ui-specification-for-registration-client - this is more human readable format and contains details in tabular format as well as description on usage as to how to use the UI components
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)